cloud/main.js

```diff
@@ -1,129 +0,0 @@
-
-// This job updates all current records 'search' field.
```

Comment:
For whatever reason, we had both a cloud and parseComCode folder. Both Andrew and I worked out of the parseComCode folder; so the duplicate cloud folder is unnecessary.